src/data/code-sample-definitions/acs-users.yaml

```
@@ -0,0 +1,105 @@
+---
+- title: Create a new ACS user
+  description: Creates a new [ACS user](https://docs.seam.co/latest/capability-guides/access-systems/user-management).
+  request:
+    path: /acs/users/create
+    parameters:
+      acs_system_id: 123e4567-e89b-12d3-a456-426614174000
+  response:
+    body:
+      acs_user:
+        - acs_user_id: 123e4567-e89b-12d3-a456-426614174000
+        - acs_system_id: 123e4567-e89b-12d3-a456-426614174000
+        - acs_access_group_ids: [123e4567-e89b-12d3-a456-426614174000]
+        - workspace_id: 123e4567-e89b-12d3-a456-426614174000
+        - created_at: 2024-04-05T07:57:05.323Z
+        - display_name: Jane Doe
+        - full_name: Jane Doe
+        - email_address: jane@example.com
+        - phone_number: +15555550100
```

Hyphen is used to create an array of items. In the code above an array for every key value pair is created which we don't want. Let's fix it
```suggestion
        - acs_user_id: 123e4567-e89b-12d3-a456-426614174000
          acs_system_id: 123e4567-e89b-12d3-a456-426614174000
          acs_access_group_ids: [123e4567-e89b-12d3-a456-426614174000]
          workspace_id: 123e4567-e89b-12d3-a456-426614174000
          created_at: 2024-04-05T07:57:05.323Z
          display_name: Jane Doe
          full_name: Jane Doe
          email_address: jane@example.com
          phone_number: +15555550100
```

---

Thank you! But I've made them regular properties instead of adding a `-` since we want an object response (not an array)